app/stac_api/templates/admin/openlayers.js

```diff
@@ -0,0 +1,203 @@
+            {{ module }}.write_wkt(feat);
+        }
+    } else {
+        {{ module }}.write_wkt(event.feature);
+    }
+};
+// Function to clear vector features and purge wkt from div
+{{ module }}.deleteFeatures = function(){
+    {{ module }}.layers.vector.removeFeatures({{ module }}.layers.vector.features);
+    {{ module }}.layers.vector.destroyFeatures();
+};
+{{ module }}.clearFeatures = function (){
+    {{ module }}.deleteFeatures();
+    document.getElementById('{{ id }}').value = '';
+    {% localize off %}
+    {{ module }}.map.setCenter(new OpenLayers.LonLat({{ default_lon }}, {{ default_lat }}), {{ default_zoom }});
+    {% endlocalize %}
+};
+// Add Select control
+{{ module }}.addSelectControl = function(){
+    var select = new OpenLayers.Control.SelectFeature({{ module }}.layers.vector, {'toggle' : true, 'clickout' : true});
+    {{ module }}.map.addControl(select);
+    select.activate();
+};
+{{ module }}.enableDrawing = function(){
+    {{ module }}.map.getControlsByClass('OpenLayers.Control.DrawFeature')[0].activate();
+};
+{{ module }}.enableEditing = function(){
+    {{ module }}.map.getControlsByClass('OpenLayers.Control.ModifyFeature')[0].activate();
+};
+// Create an array of controls based on geometry type
+{{ module }}.getControls = function(lyr){
+    {{ module }}.panel = new OpenLayers.Control.Panel({'displayClass': 'olControlEditingToolbar'});
+    {{ module }}.controls = [new OpenLayers.Control.Navigation()];
+    if (!{{ module }}.modifiable && lyr.features.length) return;
+    if ({{ module }}.is_linestring || {{ module }}.is_generic){
+        {{ module }}.controls.push(new OpenLayers.Control.DrawFeature(lyr, OpenLayers.Handler.Path, {'displayClass': 'olControlDrawFeaturePath'}));
+    }
+    if ({{ module }}.is_polygon || {{ module }}.is_generic){
+        {{ module }}.controls.push(new OpenLayers.Control.DrawFeature(lyr, OpenLayers.Handler.Polygon, {'displayClass': 'olControlDrawFeaturePolygon'}));
+    }
+    if ({{ module }}.is_point || {{ module }}.is_generic){
+        {{ module }}.controls.push(new OpenLayers.Control.DrawFeature(lyr, OpenLayers.Handler.Point, {'displayClass': 'olControlDrawFeaturePoint'}));
+    }
+    if ({{ module }}.modifiable){
+        {{ module }}.controls.push(new OpenLayers.Control.ModifyFeature(lyr, {'displayClass': 'olControlModifyFeature'}));
+    }
+};
+{{ module }}.init = function(){
+    {% block map_options %}// The options hash, w/ zoom, resolution, and projection settings.
+    var options = {
+{% autoescape off %}{% for item in map_options.items %}      '{{ item.0 }}' : {{ item.1 }}{% if not forloop.last %},{% endif %}
+{% endfor %}{% endautoescape %}    };{% endblock %}
+    // The admin map for this geometry field.
+    {% block map_creation %}
+    {{ module }}.map = new OpenLayers.Map('{{ id }}_map', options);
+    // Base Layer
+
+     // Standard WMTS options for GeoAdmin...
+    {{ module }}.layers.base = {% block base_layer %}new OpenLayers.Layer.WMTS(
```

Comment:
@procrastinatio is that ok. And do you know how to treat the referer - I get a 403 from WMTS? when developing localy